ifrs17/Import/Importers.ipynb

```diff
@@ -525,6 +525,7 @@
                 "\n              ivs.Where(iv => Math.Abs(iv.Value) < Precision).Intersect(bestEstimateIvs, EqualityComparer<IfrsVariable>.Instance).Select(iv => iv with {Value = 0.0}).ToArray());",
                 "\n    }",
                 "\n",
+                "\n    await workspace.DeleteAsync<IfrsVariable>(await workspace.Query<IfrsVariable>().ToArrayAsync()); //Clean parsed IfrsVariable",
```

Comment:
I think it is more efficient to do 
Workspace.Reset(x => x.ResetType<IfrsVariable>())